docs/source/rendering_menus/template_tag_reference.rst

```diff
@@ -96,6 +96,8 @@ The tag will attempt to add 'active' and 'ancestor' CSS classes to the menu item
 
 You can change the CSS class strings used to indicate 'active' and 'ancestor' statuses by utilising the :ref:`ACTIVE_CLASS` and :ref:`ACTIVE_ANCESTOR_CLASS` settings.
 
+By default, custom links will perform a strict comparison of the ``custom_link`` path and the current path. In this case a ``custom_link`` will never render as an 'ancestor' of a url. You can override this behavior by setting :ref:``CUSTOM_URL_SMART_ACTIVE_CLASSES`` to ``True``.
+
```

Comment:
@inostia Could we change this to...

```By default, menu items linking to custom URLs will also be attributed with the 'active' class if their ``link_url`` value  matches the path of the current request _exactly_. However, there is now an alternative solution available, which you can enable by setting :ref:``CUSTOM_URL_SMART_ACTIVE_CLASSES`` to ``True`` in your project's settings.```

I think it will be better to cover off the details of the new behaviour under the documentation for the setting, and in the release notes (which I'll put together at a later time)